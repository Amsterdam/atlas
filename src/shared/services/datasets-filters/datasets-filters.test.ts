import getDatasetFilters from './datasets-filters'

describe('fetchApiSpecification', () => {
  const state = {
    user: {
      accessToken: '',
    },
  }
  // @ts-ignore
  global.reduxStore = {
    getState: () => state,
  }

  it('should return the correct data', async () => {
    const result = await getDatasetFilters()
    expect(result).toMatchSnapshot()
  })
})
