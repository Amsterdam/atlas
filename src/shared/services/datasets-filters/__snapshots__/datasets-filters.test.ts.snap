// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`fetchApiSpecification should return the correct data 1`] = `
Object {
  "accrualPeriodicities": Array [
    Object {
      "id": "unknown",
      "label": "onbekend",
    },
    Object {
      "id": "realtime",
      "label": "continu",
    },
    Object {
      "id": "day",
      "label": "dagelijks",
    },
    Object {
      "id": "2pweek",
      "label": "twee keer per week",
    },
    Object {
      "id": "week",
      "label": "wekelijks",
    },
    Object {
      "id": "2weeks",
      "label": "tweewekelijks",
    },
    Object {
      "id": "month",
      "label": "maandelijks",
    },
    Object {
      "id": "quarter",
      "label": "eens per kwartaal",
    },
    Object {
      "id": "2pyear",
      "label": "halfjaarlijks",
    },
    Object {
      "id": "year",
      "label": "jaarlijks",
    },
    Object {
      "id": "2years",
      "label": "tweejaarlijks",
    },
    Object {
      "id": "4years",
      "label": "vierjaarlijks",
    },
    Object {
      "id": "5years",
      "label": "vijfjaarlijks",
    },
    Object {
      "id": "10years",
      "label": "tienjaarlijks",
    },
    Object {
      "id": "reg",
      "label": "regelmatig",
    },
    Object {
      "id": "irreg",
      "label": "onregelmatig",
    },
    Object {
      "id": "req",
      "label": "op afroep",
    },
    Object {
      "id": "other",
      "label": "anders",
    },
  ],
  "distributionTypes": Array [
    Object {
      "id": "api",
      "label": "API/Service",
    },
    Object {
      "id": "file",
      "label": "Bestand",
    },
    Object {
      "id": "web",
      "label": "Website",
    },
  ],
  "formatTypes": Array [
    Object {
      "id": "text/csv",
      "label": "CSV",
    },
    Object {
      "id": "application/vnd.openxmlformats-officedocument.wordprocessingml.document",
      "label": "DOCX",
    },
    Object {
      "id": "application/vnd.geo+json",
      "label": "GeoJSON",
    },
    Object {
      "id": "application/gml+xml",
      "label": "GML",
    },
    Object {
      "id": "text/html",
      "label": "HTML",
    },
    Object {
      "id": "application/json",
      "label": "JSON",
    },
    Object {
      "id": "application/pdf",
      "label": "PDF",
    },
    Object {
      "id": "image/png",
      "label": "PNG",
    },
    Object {
      "id": "application/x-zipped-shp",
      "label": "SHP",
    },
    Object {
      "id": "application/vnd.ms-excel",
      "label": "XLS",
    },
    Object {
      "id": "application/vnd.openxmlformats-officedocument.spreadsheetml.sheet",
      "label": "XLSX",
    },
    Object {
      "id": "application/xml",
      "label": "XML",
    },
    Object {
      "id": "application/octet-stream",
      "label": "Anders",
    },
  ],
  "groupTypes": Array [
    Object {
      "id": "bestuur",
      "label": "Bestuur",
    },
    Object {
      "id": "bevolking",
      "label": "Bevolking",
    },
    Object {
      "id": "cultuur-en-recreatie",
      "label": "Cultuur en recreatie",
    },
    Object {
      "id": "duurzaamheid-en-milieu",
      "label": "Duurzaamheid en milieu",
    },
    Object {
      "id": "economie-en-toerisme",
      "label": "Economie en toerisme",
    },
    Object {
      "id": "onderwijs-en-wetenschap",
      "label": "Onderwijs en wetenschap",
    },
    Object {
      "id": "openbare-orde-en-veiligheid",
      "label": "Openbare orde en veiligheid",
    },
    Object {
      "id": "ruimte-en-topografie",
      "label": "Ruimte en topografie",
    },
    Object {
      "id": "verkeer",
      "label": "Verkeer",
    },
    Object {
      "id": "werk-en-sociale-zekerheid",
      "label": "Werk en sociale zekerheid",
    },
    Object {
      "id": "wonen",
      "label": "Wonen",
    },
    Object {
      "id": "zorg-en-welzijn",
      "label": "Zorg en welzijn",
    },
  ],
  "languages": Array [
    Object {
      "id": "nl",
      "label": "Nederlands",
    },
    Object {
      "id": "en",
      "label": "Engels",
    },
  ],
  "licenseTypes": Array [
    Object {
      "id": "cc-by",
      "label": "Creative Commons, Naamsvermelding",
    },
    Object {
      "id": "cc-by-nc",
      "label": "Creative Commons, Naamsvermelding, Niet-Commercieel",
    },
    Object {
      "id": "cc-by-nc-nd",
      "label": "Creative Commons, Naamsvermelding, Niet-Commercieel, Geen Afgeleide Werken",
    },
    Object {
      "id": "cc-by-nc-sa",
      "label": "Creative Commons, Naamsvermelding, Niet-Commercieel, Gelijk Delen",
    },
    Object {
      "id": "cc-by-nd",
      "label": "Creative Commons, Naamsvermelding, Geen Afgeleide Werken",
    },
    Object {
      "id": "cc-by-sa",
      "label": "Creative Commons, Naamsvermelding, Gelijk Delen",
    },
    Object {
      "id": "cc-nc",
      "label": "Creative Commons, Niet-Commercieel",
    },
    Object {
      "id": "cc-zero",
      "label": "Publiek Domein",
    },
    Object {
      "id": "other-open",
      "label": "Anders, Open",
    },
    Object {
      "id": "other-by",
      "label": "Anders, Naamsvermelding",
    },
    Object {
      "id": "other-nc",
      "label": "Anders, Niet Commercieel",
    },
    Object {
      "id": "other-not-open",
      "label": "Anders, Niet Open",
    },
    Object {
      "id": "unspec",
      "label": "Niet gespecificeerd",
    },
  ],
  "ownerTypes": Array [
    Object {
      "id": "Amsterdam Economic Board",
      "label": "Amsterdam Economic Board",
    },
    Object {
      "id": "Amsterdam Marketing",
      "label": "Amsterdam Marketing",
    },
    Object {
      "id": "Amsterdam Museum",
      "label": "Amsterdam Museum",
    },
    Object {
      "id": "Athlon Car Lease",
      "label": "Athlon Car Lease",
    },
    Object {
      "id": "Brandweer Amsterdam-Amstelland",
      "label": "Brandweer Amsterdam-Amstelland",
    },
    Object {
      "id": "CBS",
      "label": "CBS",
    },
    Object {
      "id": "CIBG",
      "label": "CIBG",
    },
    Object {
      "id": "Cliëntenbelang",
      "label": "Cliëntenbelang",
    },
    Object {
      "id": "Cultuurcompagnie Noord-Holland",
      "label": "Cultuurcompagnie Noord-Holland",
    },
    Object {
      "id": "GGD Amsterdam",
      "label": "GGD Amsterdam",
    },
    Object {
      "id": "GOVI",
      "label": "GOVI",
    },
    Object {
      "id": "Gemeente Amsterdam",
      "label": "Gemeente Amsterdam",
    },
    Object {
      "id": "Gemeente Amsterdam - Directie Wonen - Afdeling Projectbureau Sanering Verkeerslawaai (PSV)",
      "label": "Gemeente Amsterdam - Directie Wonen - Afdeling Projectbureau Sanering Verkeerslawaai (PSV)",
    },
    Object {
      "id": "Gemeente Amsterdam, Afval en Grondstoffen",
      "label": "Gemeente Amsterdam, Afval en Grondstoffen",
    },
    Object {
      "id": "Gemeente Amsterdam, Basisinformatie",
      "label": "Gemeente Amsterdam, Basisinformatie",
    },
    Object {
      "id": "Gemeente Amsterdam, Bestuur en Organisatie",
      "label": "Gemeente Amsterdam, Bestuur en Organisatie",
    },
    Object {
      "id": "Gemeente Amsterdam, Economie",
      "label": "Gemeente Amsterdam, Economie",
    },
    Object {
      "id": "Gemeente Amsterdam, Grond en Ontwikkeling",
      "label": "Gemeente Amsterdam, Grond en Ontwikkeling",
    },
    Object {
      "id": "Gemeente Amsterdam, Monumenten en Archeologie",
      "label": "Gemeente Amsterdam, Monumenten en Archeologie",
    },
    Object {
      "id": "Gemeente Amsterdam, Onderwijs, Jeugd en Zorg",
      "label": "Gemeente Amsterdam, Onderwijs, Jeugd en Zorg",
    },
    Object {
      "id": "Gemeente Amsterdam, Onderzoek, Informatie en Statistiek",
      "label": "Gemeente Amsterdam, Onderzoek, Informatie en Statistiek",
    },
    Object {
      "id": "Gemeente Amsterdam, Projectmanagementbureau",
      "label": "Gemeente Amsterdam, Projectmanagementbureau",
    },
    Object {
      "id": "Gemeente Amsterdam, Ruimte en Duurzaamheid",
      "label": "Gemeente Amsterdam, Ruimte en Duurzaamheid",
    },
    Object {
      "id": "Gemeente Amsterdam, Sport en Bos",
      "label": "Gemeente Amsterdam, Sport en Bos",
    },
    Object {
      "id": "Gemeente Amsterdam, Stadsarchief",
      "label": "Gemeente Amsterdam, Stadsarchief",
    },
    Object {
      "id": "Gemeente Amsterdam, Stadsdeel Centrum",
      "label": "Gemeente Amsterdam, Stadsdeel Centrum",
    },
    Object {
      "id": "Gemeente Amsterdam, Stadsdeel West",
      "label": "Gemeente Amsterdam, Stadsdeel West",
    },
    Object {
      "id": "Gemeente Amsterdam, Verkeer en Openbare Ruimte",
      "label": "Gemeente Amsterdam, Verkeer en Openbare Ruimte",
    },
    Object {
      "id": "Gemeente Amsterdam, Wonen",
      "label": "Gemeente Amsterdam, Wonen",
    },
    Object {
      "id": "Gemeente Amsterdam, programma Afval Keten",
      "label": "Gemeente Amsterdam, programma Afval Keten",
    },
    Object {
      "id": "Gemeente Amsterdam, stadsdeel Zuidoost",
      "label": "Gemeente Amsterdam, stadsdeel Zuidoost",
    },
    Object {
      "id": "JeKuntMeer.nl",
      "label": "JeKuntMeer.nl",
    },
    Object {
      "id": "KNMI",
      "label": "KNMI",
    },
    Object {
      "id": "Kadaster",
      "label": "Kadaster",
    },
    Object {
      "id": "Landelijk Register Kinderopvang en Peuterspeelzalen",
      "label": "Landelijk Register Kinderopvang en Peuterspeelzalen",
    },
    Object {
      "id": "Liander",
      "label": "Liander",
    },
    Object {
      "id": "Metro en Tram",
      "label": "Metro en Tram",
    },
    Object {
      "id": "Ministerie van OCW",
      "label": "Ministerie van OCW",
    },
    Object {
      "id": "Nationale Databank Wegverkeergegevens",
      "label": "Nationale Databank Wegverkeergegevens",
    },
    Object {
      "id": "Open Cultuur Data",
      "label": "Open Cultuur Data",
    },
    Object {
      "id": "Rijksdienst voor Cultureel Erfgoed",
      "label": "Rijksdienst voor Cultureel Erfgoed",
    },
    Object {
      "id": "Rijksdienst voor Ondernemend Nederland",
      "label": "Rijksdienst voor Ondernemend Nederland",
    },
    Object {
      "id": "Rijksmuseum Amsterdam",
      "label": "Rijksmuseum Amsterdam",
    },
    Object {
      "id": "Rijkswaterstaat",
      "label": "Rijkswaterstaat",
    },
    Object {
      "id": "UWV",
      "label": "UWV",
    },
    Object {
      "id": "Waag Society",
      "label": "Waag Society",
    },
  ],
  "resourceTypes": Array [
    Object {
      "id": "data",
      "label": "Data",
    },
    Object {
      "id": "doc",
      "label": "Documentatie",
    },
    Object {
      "id": "vis",
      "label": "Visualisatie",
    },
    Object {
      "id": "app",
      "label": "Voorbeeldtoepassing",
    },
  ],
  "serviceTypes": Array [
    Object {
      "id": "atom",
      "label": "REST: Atom feed",
    },
    Object {
      "id": "rest",
      "label": "REST: overig",
    },
    Object {
      "id": "csw",
      "label": "CSW",
    },
    Object {
      "id": "wcs",
      "label": "WCS",
    },
    Object {
      "id": "wfs",
      "label": "WFS",
    },
    Object {
      "id": "wms",
      "label": "WMS",
    },
    Object {
      "id": "wmts",
      "label": "WMTS",
    },
    Object {
      "id": "soap",
      "label": "SOAP",
    },
    Object {
      "id": "other",
      "label": "Anders",
    },
  ],
  "spatialUnits": Array [
    Object {
      "id": "na",
      "label": "Geen geografie",
    },
    Object {
      "id": "specific",
      "label": "Specifieke punten/vlakken/lijnen",
    },
    Object {
      "id": "nation",
      "label": "Land",
    },
    Object {
      "id": "region",
      "label": "Regio",
    },
    Object {
      "id": "city",
      "label": "Gemeente",
    },
    Object {
      "id": "district",
      "label": "Stadsdeel",
    },
    Object {
      "id": "area",
      "label": "Gebied",
    },
    Object {
      "id": "borrow",
      "label": "Wijk",
    },
    Object {
      "id": "neighborhood",
      "label": "Buurt",
    },
    Object {
      "id": "block",
      "label": "Bouwblok",
    },
    Object {
      "id": "zip4",
      "label": "Postcode (4 cijfers)",
    },
    Object {
      "id": "zip5",
      "label": "Postcode (4 cijfers, 1 letter)",
    },
    Object {
      "id": "zip6",
      "label": "Postcode (4 cijfers, 2 letters)",
    },
    Object {
      "id": "other",
      "label": "anders",
    },
  ],
  "statusTypes": Array [
    Object {
      "id": "in_onderzoek",
      "label": "In onderzoek",
    },
    Object {
      "id": "niet_beschikbaar",
      "label": "Niet beschikbaar",
    },
    Object {
      "id": "beschikbaar",
      "label": "Beschikbaar",
    },
    Object {
      "id": "gepland",
      "label": "Gepland",
    },
  ],
  "temporalUnits": Array [
    Object {
      "id": "na",
      "label": "Geen tijdseenheid",
    },
    Object {
      "id": "realtime",
      "label": "Realtime",
    },
    Object {
      "id": "minutes",
      "label": "Minuten",
    },
    Object {
      "id": "hours",
      "label": "Uren",
    },
    Object {
      "id": "parttime",
      "label": "Dagdelen",
    },
    Object {
      "id": "days",
      "label": "Dagen",
    },
    Object {
      "id": "weeks",
      "label": "Weken",
    },
    Object {
      "id": "months",
      "label": "Maanden",
    },
    Object {
      "id": "quarters",
      "label": "Kwartalen",
    },
    Object {
      "id": "years",
      "label": "Jaren",
    },
    Object {
      "id": "other",
      "label": "anders",
    },
  ],
}
`;
